src/libsync/determineauthtypejob2.cpp

```diff
@@ -0,0 +1,49 @@
+#include "determineauthtypejob2.h"
```

Comment:
Please add new classes to a subfolder, we will probably still need to add the to the src from this directories CMakeLists.txt as add_target_src dosn't support relative paths, but it will give the project at least some mental sturcture.